### PR TITLE
[DOC] Deploy an EC2 instance with Meshery

### PR DESCRIPTION
**Pull request by alero-awani**
**Notes for Reviewers**
Links to published catalog designs need to be added.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit cb537b88e435f435c6b68369efb0ef4044b008fb at: https://672379f037cfc077029a9873--meshery-docs-preview.netlify.app

File: docs/_data/toc.yml [hunk 1]
```diff
@@ -160,21 +160,23 @@
...
-        url: guides/tutorials/wordpress-mysql-persistentvolume
+        - title: Exploring Kubernetes CronJobs
+          url: guides/tutorials/exploring-kubernetes-cronjobs
+        - title: Understanding Kubernetes ConfigMaps and Secrets with Meshery
+          url: guides/tutorials/kubernetes-configmaps-secrets
+        - title: Exploring Kubernetes Pods with Meshery
+          url: guides/tutorials/kubernetes-pods
+        - title: Deploying WordPress and MySQL with Persistent Volumes with Meshery
+          url: guides/tutorials/wordpress-mysql-persistentvolume
+        - title: Deploy an EC2 Instance with Meshery
```
Comment (leecalcote) [inline docs/_data/toc.yml:178]:
```suggestion
        - title: Deploy an AWS EC2 Instance with Meshery
```

File: docs/_data/toc.yml [hunk 2]
```diff
@@ -160,21 +160,23 @@
...
+          url: guides/tutorials/deploy-an-ec2-instance-with-meshery
```
Comment (leecalcote) [inline docs/_data/toc.yml:179]:
```suggestion
          url: guides/tutorials/deploy-an-aws-ec2-instance-with-meshery
```

File: docs/pages/guides/tutorials/deploy-an-ec2-instance-with-meshery.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
+---
+layout: default
+title: Deploy an EC2 Instance with Meshery
+abstract: "Learn how to deploy and manage an EC2 instance through Kubernetes with Meshery, utilizing AWS Controllers for Kubernetes (ACK) to enhance cloud resource management"
+permalink: guides/tutorials/deploy-an-ec2-instance-with-meshery
+model: aws
```
Comment (leecalcote) [inline docs/pages/guides/tutorials/deploy-an-ec2-instance-with-meshery.md:6]: This `model` is inaccurate, isn't it, @Alero-Awani?

File: docs/pages/guides/tutorials/deploy-an-ec2-instance-with-meshery.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
+---
+layout: default
+title: Deploy an EC2 Instance with Meshery
+abstract: "Learn how to deploy and manage an EC2 instance through Kubernetes with Meshery, utilizing AWS Controllers for Kubernetes (ACK) to enhance cloud resource management"
+permalink: guides/tutorials/deploy-an-ec2-instance-with-meshery
```
Comment (leecalcote) [inline docs/pages/guides/tutorials/deploy-an-ec2-instance-with-meshery.md:5]:
```suggestion
title: Deploy an AWS EC2 Instance with Meshery
abstract: "Learn how to deploy and manage an AWS EC2 instance through Kubernetes with Meshery, utilizing AWS Controllers for Kubernetes (ACK) to enhance cloud resource management"
permalink: guides/tutorials/deploy-an-aws-ec2-instance-with-meshery
```

File: docs/pages/guides/tutorials/deploy-an-ec2-instance-with-meshery.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+model: aws
+kind: EC2
+type: guides
+category: tutorials
+language: en
+list: include
+abstract: "Learn how to deploy and manage an EC2 instance through Kubernetes with Meshery, utilizing AWS Controllers for Kubernetes (ACK) to enhance cloud resource management"
```
Comment (leecalcote) [inline docs/pages/guides/tutorials/deploy-an-ec2-instance-with-meshery.md:12]:
```suggestion
```

remove duplicate

File: docs/pages/guides/tutorials/deploy-an-ec2-instance-with-meshery.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+---
+
+### Introduction
+
+Meshery is a powerful multi-cloud management platform that can allow you deploy and manage AWS resources, such as EC2 instances, within a Kubernetes environment. It does this by leveraging [AWS Controllers for Kubernetes (ACK)](https://aws.amazon.com/blogs/containers/aws-controllers-for-kubernetes-ack/). ACK provides a bridge between Kubernetes and AWS services, enabling developers to manage AWS resources directly from within their Kubernetes clusters using the Kubectl CLI. Meshery takes this functionality further by providing **Kanvas**, an intuitive, visual interface that simplifies the deployment process.
```
Comment (leecalcote) [inline docs/pages/guides/tutorials/deploy-an-ec2-instance-with-meshery.md:17]:
```suggestion
Meshery is a powerful multi-cloud management platform that enables you to configure, deploy, and manage AWS resources, such as EC2 instances. It does this by leveraging [AWS Controllers for Kubernetes (ACK)](https://aws.amazon.com/blogs/containers/aws-controllers-for-kubernetes-ack/). ACK provides a bridge between Kubernetes and AWS services, enabling developers to manage AWS resources directly from within their Kubernetes clusters using the Kubectl CLI. Meshery takes this functionality further by providing **Kanvas**, an intuitive, visual interface that simplifies the deployment process.
```

The EC2 instances don't run inside of Kubernetes, so I'm removing ", within a Kubernetes environment".

File: docs/pages/guides/tutorials/deploy-an-ec2-instance-with-meshery.md [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+
+With **Kanvas**, Meshery provides an alternative to command-line tools like kubectl by offering a more visual approach. Once you connect your Kubernetes cluster to Meshery, you can configure, deploy, and manage AWS resources directly from the Kanvas interface, making deployments more intuitive and providing a collaborative environment for team-based deployments.
```
Comment (leecalcote) [inline docs/pages/guides/tutorials/deploy-an-ec2-instance-with-meshery.md:19]:
```suggestion
Meshery has a number of extensions, adapters, and plugins. In this tutorial, we will use the [Kanvas]({{site.baseurl}}/extensibility/extensions/kanvas) extension to provide an intuitive, visual experience for configuring and deploying an AWS EC2 instance. Among other aspects, Kanvas provides an alternative to command-line tools like `kubectl` by offering infrastructure as design. Once you connect your Kubernetes cluster to Meshery, you can configure, deploy, and manage AWS resources directly from the Kanvas interface, making deployments more intuitive and collaborative 
```

File: docs/pages/guides/tutorials/deploy-an-ec2-instance-with-meshery.md [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+
+In this guide, you’ll explore how to deploy AWS resources using Kanvas, including setup and architecture details. This guide also covers how to access pre-configured designs from Meshery’s catalog and demonstrates how to visualize deployed resources using Meshery’s operator mode, offering a comprehensive understanding of AWS resource management within **Kanvas**
```
Comment (leecalcote) [inline docs/pages/guides/tutorials/deploy-an-ec2-instance-with-meshery.md:21]:
```suggestion
In this guide, you’ll explore how to deploy AWS resources, including setup and architecture details. This guide also covers how to access pre-configured designs from [Meshery Catalog](https://meshery.io/catalog) and demonstrates how to visualize deployed resources using Kanvas' operator mode, offering a comprehensive understanding of AWS resource management.
```

File: docs/pages/guides/tutorials/deploy-an-ec2-instance-with-meshery.md [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+
+### Prerequisites
+
+1. [Self Hosted Meshery Installation](https://docs.meshery.io/installation).
+1. **AWS Account**: An active AWS account is required for accessing AWS resources.
+1. **Kubernetes Cluster**: A running Kubernetes cluster, such as Minikube or EKS.
+
+### Table of Content
```
Comment (leecalcote) [inline docs/pages/guides/tutorials/deploy-an-ec2-instance-with-meshery.md:29]:
```suggestion
### Table of Contents
```

**Review by leecalcote — APPROVED** [review]

**Comment by leecalcote** [comment]
Good work, @Alero-Awani! 😄 One or more of the published designs can be embedded in this tutorial and/or the can be linked to.

File: docs/pages/guides/tutorials/aws-controllers/controller-chart.png
[the remark this answers is not in the document]
Comment (Alero-Awani) [reply docs/pages/guides/tutorials/aws-controllers/controller-chart.png:1]: I'll replace this image with the new design

**Pull request merged**